codalab/worker_manager/worker_manager.py

```diff
@@ -113,7 +113,7 @@ def run_one_iteration(self):
         if os.environ.get('CODALAB_USERNAME') != "codalab":
             keywords += [".mine"]
         if self.args.worker_tag_exclusive and self.args.worker_tag:
-            keywords += ["request_queue=" + self.args.worker_tag]
+            keywords += ["request_queue=%s,tag=%s" % (self.args.worker_tag, self.args.worker_tag)]
```

Comment:
Could you leave a comment here for why we need both and also say that in the future, we just need `request_queue`?

Reply:
sure. also worth noting that there's a corresponding piece of code in bundle manager that treats `tag=` as an optional but matching prefix there. it's probably best to keep these two consistent ie. if this is removed then so should that bit.

Reply:
done

Reply:
Just realized that this is matching <worker_tag> or tag=<worker_tag>, which is the natural way to read this expression and makes more sense.  Could you add a quick comment?

Reply:
oh, sure